utils/index.spec.js

```diff
@@ -36,6 +37,42 @@ describe('Test SINFilter', () => {
   })
 })
 
+describe('Test currencyFilter', () => {
```

Comment:
Let's write a few tests for `postAmount` as well to test the repeating spaces thing.